jobs/config.json

```diff
@@ -3699,6 +3699,23 @@
       "generated"
     ]
   },
+  "ci-kubernetes-e2e-gce-min-node-permissions": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-min-node-permissions.env",
+      "--extract=ci/latest",
+      "--gcp-project=k8s-min-node-permissions",
```

Comment:
does this project has some special quota/IAM?

Reply:
It has the min-node-permissions@k8s-min-node-permissions.iam.gserviceaccount.com service account configured.